src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources_de.properties

```diff
@@ -30,46 +30,45 @@ msi.bundler.name=MSI-Installationsprogrammpackage
 
 param.menu-group.default=Unbekannt
 
-resource.executable-properties-template=Vorlage für das Erstellen der ausführbaren Eigenschaftendatei
-resource.setup-icon=Symbol für Dialogfeld "Setup"
-resource.post-app-image-script=Auszuführendes Skript nach dem Auffüllen des Anwendungsimages
-resource.post-msi-script=Auszuführendes Skript nach dem Erstellen der MSI-Datei für das EXE-Installationsprogramm
+resource.executable-properties-template=Vorlage f�r das Erstellen der ausf�hrbaren Eigenschaftendatei
```

Comment:
Looks good. Do you know why this file show changes? Maybe you saved with wrong encoding or line ending?

Reply:
Ugh, the diff looked right in TortoiseGit. I'll fix it